### DISCUSSION TITLE
VBA & References / Office 16.0 Object Library

### DISCUSSION THREAD

**Discussion by m0styr** [dfinke/ImportExcel#1593] [Q&A]
Hi **ImportExcel** community,
please, when I'm creating *.xlsm file with VBA code via **ImportExcel**, then given file has available only two references: _Visual Basic For Applications_ and _Microsoft Excel 16.0 Object Library_.
I would like to add _Microsoft Office 16.0 Object Library_ to be able use select file dialog in VBA (Application.fileDialog(msoFileDialogOpen)).
Is it possible in **ImportExcel**?
Thanks in advance for info.

![image](https://github.com/dfinke/ImportExcel/assets/48366970/a22a95b6-a720-480c-8621-4fdf8a505aba)

**Comment by dfinke** [comment]
Thanks for using the module @m0styr. I have not tried to do that. I suspect it cannot be done, but that needs to be verified.
